src/app/modules/submission/email-submission/email-submission.util.ts

```diff
@@ -618,3 +641,62 @@ export class SubmissionEmailObj {
     )
   }
 }
+
+/**
+ * @param responses answerArray from address field
+ * @returns human-readable format of address, similar to handleAddressResponseDisplay (mutations.ts)
```

Comment:
If the function's input and output are the same, could we move them to a shared utils for better reuse?